integrations/pagerduty.md

```diff
@@ -31,8 +31,9 @@ If want to use a PagerDuty email target, consider the **Open a new incident only
 ### Step 1. Retrieve Your PagerDuty API Integration Key or Email Address
 
 1. Log into your PagerDuty account.
-1. Click the **Services** tab.
-1. In the Integrations column, click the **API** or **Email** link.
+1. Click the **Services** tab, and choose the service you want to retrieve the key or email address.
+{% include image.md width="50" src="images/pagerduty_services.png" %}
+1. In the Integrations tab, expand the integration to view **API** or **Email** link.
```

Comment:
On the Integrations tab, expand the integration to view the **API** or **Email** link.